src/Utils.js

```diff
@@ -0,0 +1,48 @@
+import $ from 'jquery';
+import { proxy, minYear, maxYear, defaultYear } from './config';
+
+function parseHtml(html) {
+  let parser = new DOMParser();
+  return parser.parseFromString(html, 'text/html');
+}
+
+exports.fetchDocument = function(url) {
+  return fetch(url)
+    .then((response) => response.json())
+    .then(html => parseHtml(html))
+}
+
+exports.parseInput = function() {
+  let inputYear = $('#inputYear').val();
+  let errorMess = $('.errorMess');
```

Comment:
If you really, badly, want to use shorter form then you should use: `errMsg` 😉 
No _mess_ is allowed...